pkg/config/net.go

```diff
@@ -2,12 +2,48 @@ package config
 
 import (
 	"fmt"
+	"io/ioutil"
 	"net"
 	"strings"
 
 	"github.com/openshift/baremetal-runtimecfg/pkg/utils"
 )
 
+const (
+	NodeIpIpV6File = "/run/nodeip-configuration/ipv6"
+	NodeIpIpV4File = "/run/nodeip-configuration/ipv4"
+)
+
+// Return ip from primaryIp file if file and ip exists and readable
+// In case of error return empty string
+func GetIpFromFile(filePath string) (net.IP, error) {
+	b, err := ioutil.ReadFile(filePath)
+	if err != nil {
+		log.WithError(err).Infof("Failed to read ip from file %s", filePath)
+		return nil, err
+	}
+	ip := net.ParseIP(string(b))
+	if ip == nil {
+		msg := fmt.Sprintf("Failed to parse ip from file %s", filePath)
```

Comment:
nit: other places are using `log.WithError()` which looks like avoid explicit fmt.Sprint() if I got that correctly. 
Should we use it here too?

Reply:
There is no error here to print parse ip function so I create new one a don't want to write same string  twice

Reply:
Just to clarify, I was referring to line `22` as reference.